Chetna_Mongmaw/htmltask/linked.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="UTF-8" />
+<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<meta name="description" content="my profile"/>
+<meta name="author" content="chetna"/>
+<link rel="shortcut icon" type="image/x-icon" href="eco.ico" />
+</head>
+<body>
+<form action="">
+<div ">
+<h2>Greetings!!</h2>
+<br/>
+<br/>
+Name: &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp <input type="text" name="uname" /><br/><br/>
```

Comment:
&nbsp needs to end with a semi-colon it should be &nbsp;

Reply:
also get rid of &nbsp; this is degrading readability,
make visible adjustment using CSS syling

Reply:
corrected